### DISCUSSION TITLE
RPSL Syntax

### DISCUSSION THREAD

**Discussion by rubenskuhl** [irrdnet/irrd#908] [Q&A] [answered]
Hi there. My doc reading foo is failing me, but I swear there was a query to IRRd where it responds with all RPSL syntax definitions. Does someone remember which query it was ?

**Comment by dhutchin-ctl** [comment, marked as answer]
You mean like:
·         -t <object-class> returns the template for a particular object class.
???

From: Rubens Kuhl ***@***.***>
Sent: Tuesday, February 06, 2024 07:26
To: irrdnet/irrd ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [irrdnet/irrd] RPSL Syntax (Discussion #908)


CAUTION: This email originated outside of Lumen Technologies. Do not click links or open attachments unless you recognize the sender and know the content is safe.


Hi there. My doc reading foo is failing me, but I swear there was a query to IRRd where it responds with all RPSL syntax definitions. Does someone remember which query it was ?

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/irrdnet/irrd/discussions/908__;!!CdLFVIQ!TgDSp6udzj5hW-FiGMLplCndVAcW29JhIgFBDVTKMIojO5ywCuCHXWebMeLZn0-MFfEmnk1-Ol3jQZtIvRzPr7AXMF4dqg$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AUEB52ECDV2IRV7Z7KYKGJLYSI4QFAVCNFSM6AAAAABC4C2E22VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGE4DOMRSGA__;!!CdLFVIQ!TgDSp6udzj5hW-FiGMLplCndVAcW29JhIgFBDVTKMIojO5ywCuCHXWebMeLZn0-MFfEmnk1-Ol3jQZtIvRzPr7CMw3v2-g$>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.******@***.***>>

----------------------------------------------------------------------
This communication is the property of Lumen Technologies and may contain confidential or privileged information. Unauthorized use of this communication is strictly prohibited and may be unlawful. If you have received this communication in error, please immediately notify the sender by reply e-mail and destroy all copies of the communication and any attachments.

**Reply by rubenskuhl** [reply]
The one that exists in my head has all that content for all object classes... but that will do fine.
